packages/angular/src/selector/selector.component.ts

```diff
@@ -385,25 +397,42 @@ export class SelectorComponent extends RootComponent { // implements OnInit, OnC
         _.forEach(models, (model) => model.priority = priority++)
     }
 
-    getSVG(url: string) {
-        console.log('get:', url)
-        return this.iconRegistry.getSvgIconFromUrl(
-            this.sanitizer.bypassSecurityTrustResourceUrl(url)
-        )
+    getSvg(url: string, options?: IconOptions): Observable<string> {
```

Comment:
This returns an observable that pushes the `uid` of the SVG after the XHR completes.  This is for use with an `AsyncPipe` in a `<mat-icon>`.  I will add this as a comment in the code as well.